blob/gcsblob/gcsblob_test.go

```diff
@@ -16,177 +16,40 @@ package gcsblob
 
 import (
 	"context"
-	"flag"
-	"fmt"
-	"io"
-	"net/http"
-	"strings"
 	"testing"
 
-	"cloud.google.com/go/storage"
-	"github.com/dnaeon/go-vcr/recorder"
-	"github.com/google/go-cloud/gcp"
-	"github.com/google/go-cloud/internal/testing/replay"
+	"github.com/google/go-cloud/blob"
+	"github.com/google/go-cloud/blob/drivertest"
 	"github.com/google/go-cloud/internal/testing/setup"
 	"google.golang.org/api/googleapi"
-	"google.golang.org/api/option"
 )
 
-const bucketPrefix = "go-cloud"
-
-var projectID = flag.String("project", "", "GCP project ID (string, not project number) to run tests against")
-
-func TestNewBucketNaming(t *testing.T) {
-	tests := []struct {
-		name, bucketName string
-		wantErr          bool
-	}{
-		{
-			name:       "A good bucket name should pass",
-			bucketName: "bucket-name",
-		},
-		{
-			name:       "A name with leading digits should pass",
-			bucketName: "8ucket_nam3",
-		},
-		{
-			name:       "A name with a leading underscore should fail",
-			bucketName: "_bucketname_",
-			wantErr:    true,
-		},
-		{
-			name:       "A name with an uppercase character should fail",
-			bucketName: "bucketnameUpper",
-			wantErr:    true,
-		},
-		{
-			name:       "A name with an invalid character should fail",
-			bucketName: "bucketname?invalidchar",
-			wantErr:    true,
-		},
-		{
-			name:       "A name that's too long should fail",
-			bucketName: strings.Repeat("a", 64),
-			wantErr:    true,
-		},
-	}
-
+// bucketName records the bucket used for the last --record.
+// If you want to -use --record mode,
```

Comment:
Is it just "--record" or "-use --record"?

Reply:
Just --record; fixed.